backend/api/tests/test_permissions.py

```
@@ -0,0 +1,253 @@
+import pytest
+from django.test import RequestFactory
```

Ruff (0.8.2)

`django.test.RequestFactory` imported but unused

Remove unused import: `django.test.RequestFactory`

(F401)